### PR TITLE
Add support for --token-usage/-t flag to display token information

### LINKED ISSUE #5: [Feature] Option -t to display token information
### Description
Add a `-t` or `--token-usage` option. When the program is run with this option, display additional information about token usage.

Would love to work on this issue myself. Feel free to assign me.

### Feature Checklist
- Support `-t` option
- From the API response, read token information under the `usage` object
- Output to `stderr`
- Update help information to include this option
- Update README

### Sample Output

```bash
# Existing Output... #
----------------------------------
Completion Tokens: 17
Prompt Tokens: 57
Total Tokens: 74
```

### PR DESCRIPTION
**Pull request by theoforger**
This pull request fixes #5.

### Changes
- Function `callApi` returns `JSONObject` instead of `String`.
  - Since the program now needs more information from the API response, returning `String` won't be sufficient. Instead, the entire JSON object is returned. Data extraction is moved to the main function.
- Added local variables `inputTokens` and `outputTokens`.
  - Since the program supports multiple input files. This is needed to keep track of token information from all API calls combined.
 - Used `System.err.printil()` to output token information to `stderr`.
 - Updated `README.md` to include information of this new feature.

### CODE REVIEW

**Comment by mulla028** [comment]
Good contribution! Code works, and shows number of input tokens and the number of output tokens. I suggest you to output total amount of tokens as well, so you will follow the expected output :)

File: src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -107,18 +120,31 @@ public Integer call() throws Exception {
...
+            outputTokens += resultJSON.getJSONObject("meta").getJSONObject("tokens").getInt("output_tokens");
+        }
+
+        // If -t option is present
+        // Output token information to stderr
+        if (tokenUsage) {
+            System.err.println("\n------------------------\n" +
+                    "Token Information:\n" +
+                    "Input Tokens: " + inputTokens + "\n" +
+                    "Output Tokens: " + outputTokens);
```
Comment (mulla028) [inline src/main/java/org/example/Main.java:147]: Users also want to see the total amount of tokens

**Review by mulla028 — CHANGES_REQUESTED** [review]
It looks very good, just one little thing, add total amount of tokens.
Overall, good job!

**Comment by theoforger** [comment]
Thanks for the review!

I initially decided to not include the total token number because Cohere doesn't include that in their [API response](https://docs.cohere.com/docs/chat-api#response-structure). However, since it is requested, I will add this feature.

Good day!

**Comment by theoforger** [comment]
Done! Feel free to let me know if anything else needs to be changed

**Pull request merged**